src/main/java/gregtech/common/covers/CoverConveyor.java

```diff
@@ -469,7 +470,7 @@ public void readFromNBT(NBTTagCompound tagCompound) {
             this.isWorkingAllowed = tagCompound.getBoolean("WorkingAllowed");
         }
         if (tagCompound.hasKey("AllowManualIO")) {
-            this.allowManualImportExport = tagCompound.getBoolean("AllowManualIO");
+            this.allowManualImportExportMode = AllowManualImportExportMode.values()[tagCompound.getInteger("AllowManualIO")];
```

Comment:
i agree this is hard to understand, i used this because it is used in the filter mode selection in the filter cover.
"AllowManualIO" is what was used by the previous implementation and is a boolean. "ManualImportExportMode" , this implementation has 3 states.

Reply:
i only understood what you meant now. fixed it now